rules/bigquery_rules.yaml

```diff
@@ -15,36 +15,21 @@
 rules:
   - name: BigQuery rule to search for public datasets
     mode: blacklist
-    dataset_id: '*'
```

Comment:
Can we add a comment in this rule file saying that for the following fields, if not defined, the values will be defaulted to '*' so if user wants to define a rule on their own, they know what fields can be changed.

Reply:
Nice suggestion, done. It can be useful to have a more explicit schema to highlight these fields, filed #1962 for it.